package.json

```diff
@@ -80,6 +80,9 @@
     },
     "root": true
   },
+  "resolutions": {
+    "rollup": "npm:rollup@3.29.4"
```

Comment:
Temporary solution that fixes two issues.
1. Rollup v4 installs a binary based on `os`and `cpu`. That binary is used as part of nebulas build/serve command. But it might not match the `os/cpu` when running `nebula serve` command inside the docker container through `yarn test:local:rendering` or `yarn test:local:update:screenshots`.
2. The "watch" functionally of nebula is broken since nebula updated to rollup v4. So it does not pick up code changes.